app/build.gradle.kts

```diff
@@ -53,6 +53,15 @@ android {
     packaging {
         resources {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
+            excludes += "META-INF/LICENSE.md"
+            excludes += "META-INF/LICENSE-notice.md"
+        }
+    }
+    testOptions {
+        packagingOptions {
+            jniLibs {
+                useLegacyPackaging = true
+            }
```

Comment:
This is already in the main branch, maybe you can roll back those changes and then rebase main.

Reply:
no need to roll back, just doing the rebase should do the work :) either the changes are exactly are exactly identical and git will understand magically that it's already in main; otherwise it will let you resolve the conflict...

Reply:
Solved with the rebase